requirements.txt

```diff
@@ -0,0 +1,19 @@
+click==7.1.2
```

Comment:
Good question. Netlify's auto deployment wants package.json and requirements.txt to be in the same directory. The proper way to fix this might be to separate dependencies, build process, etc. from TerriaJS specs' ones, but I think that would be quite complicated (at least until we make TerriaJS more like a library package).